content/api/power-board.md

```diff
@@ -19,4 +19,29 @@ r.power_board.power_off()
 `power_on` is called when you setup your robot, so this doesn't need to be called manually. The ports will come on automatically as soon as your robot is ready, before the start button is pressed.
 {{% /notice %}}
 
+## Buzzing :bee:
+
+The power board has a piezo buzzer which can buzz.
+
+The `buzz` function accepts multiple parameters, depending on what you want to play. The first argument is the duration of the beep, in seconds. The later arguments are either the note you want to play, or the buzzers frequency (in Hz).
+
+Theoretically, the piezo buzzer will buzz at any provided frequency, however humans can only hear between [20Hz and 20000Hz](https://en.wikipedia.org/wiki/Hearing_range#Humans).  
+
+{{% notice info %}}
+To help prevent possible errors, you must specify which argument you're passing. Simply calling `buzz(2, 'c')` will raise a `TypeError`
```

Comment:
Should this end with a full stop? (I'm not sure if this is an intentional omission -- we seem to be missing full stops in a number of these `notice_info`s)